### ISSUE TITLE
Getting Random Nan for Loss

### ISSUE THREAD

**Issue by fatihcankurnaz** [open-mmlab/mmdetection#2675]
Hi,

I am getting random nan value for loss and loss_bbox in faster_rcnn_r50_fpn.py. To keep track of whether it is caused by my custom dataset or not, I am using 1 worker, 1 sample size, and printing the input file name. nan value for loss and loss_bbox occurs for random files(for example in first epoch nothing gives nan, then in second epoch 1 file gives nan, in third epoch, another file gives nan and so on.)

Similarly, in retinanet_r50_fpn_1x_coco.py I am getting inf for loss and loss_bbox in random points in the training.

Am I doing something wrong?

My system info:
sys.platform: linux
Python: 3.7.7 (default, May  6 2020, 10:21:04) [GCC 7.3.0]
CUDA available: True
CUDA_HOME: /usr/local/cuda
NVCC: Cuda compilation tools, release 10.2, V10.2.89
GPU 0: GeForce GTX 1080 Ti
GCC: gcc (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
PyTorch: 1.5.0
PyTorch compiling details: PyTorch built with:
  - GCC 7.3
  - C++ Version: 201402
  - Intel(R) Math Kernel Library Version 2020.0.0 Product Build 20191122 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v0.21.1 (Git Hash 7d2fd500bc78936d1d648ca713b901012f470dbc)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - NNPACK is enabled
  - CPU capability usage: NO AVX
  - CUDA Runtime 10.2
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_61,code=sm_61;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_37,code=compute_37
  - CuDNN 7.6.5
  - Magma 2.5.2
  - Build settings: BLAS=MKL, BUILD_TYPE=Release, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -fopenmp -DNDEBUG -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DUSE_INTERNAL_THREADPOOL_IMPL -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, USE_CUDA=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, USE_STATIC_DISPATCH=OFF, 

TorchVision: 0.6.0a0+82fd1c8
OpenCV: 4.2.0
MMCV: 0.5.1
MMDetection: 2.0.0+6603790
MMDetection Compiler: GCC 7.5
MMDetection CUDA Compiler: 10.2

**Comment by ZwwWayne** [comment]
Please use the `Error Template`. You might need to be more specific about the modification you have done to the codebase and the config, e.g, how many classes in your dataset, what else keys you modified in the config?

**Comment by ObKsEm** [comment]
@fatihcankurnaz Hi, have you solved the problem? I got a same after updating new version.

**Comment by fatihcankurnaz** [comment]
Hi, sorry but I could not. Because of the urgency of my project, I rolled back to version 1.0.0rc(which was the version I was using before updating), and it is working fine for me.

**Comment by ObKsEm** [comment]
> Hi, sorry but I could not. Because of the urgency of my project, I rolled back to version 1.0.0rc(which was the version I was using before updating), and it is working fine for me.

So sad, thank you.

**Comment by ZwwWayne** [comment]
Hi @ObKsEm ,
If you meet the problem and do not know how to solve it, please raise a new issue following `Error Template`.
